compute/cker/include/cker/operation/BinaryArithmeticOps.h

```diff
@@ -221,11 +221,11 @@ BinaryArithmeticOp(const BinaryArithmeticOpParam &params, const Shape &input1_sh
 template <BinaryArithmeticOpType op_type, typename T>
 inline typename std::enable_if_t<!is_quant8<T>::value && std::is_same<T, bool>::value>
 BinaryArithmeticOp(const BinaryArithmeticOpParam &params, const Shape &input1_shape,
-                   const bool *input1_data, const Shape &input2_shape, const bool *input2_data,
-                   const Shape &output_shape, bool *output_data)
+                   const T *input1_data, const Shape &input2_shape, const T *input2_data,
+                   const Shape &output_shape, T *output_data)
 {
   reference::BinaryArithmeticOp(params, input1_shape, input1_data, input2_shape, input2_data,
-                                output_shape, output_data, GetBinaryArtithmeticFn<op_type, bool>());
+                                output_shape, output_data, GetBinaryArtithmeticFn<op_type, T>());
```

Comment:
I have a concern about using bool type array because tensorflow lite and our runtime uses 8bit for bool type, but c++'s sizeof(bool) is not restricts as 8bit. I think it has similar risky with `int`.

Reply:
Although c++'s `sizeof(bool)` is not restricted as 1 byte, most of `sizeof(bool)` seems to be 1 bytes. How about throw an exception when `sizeof(bool)` is not 1 byte? The place is where data is converted in onert?

Reply:
Why don't we use `static_assert`? 

ex)
https://github.com/Samsung/ONE/blob/8180a7ce28f147ea43fa143840eb23c3d0f5708a/compiler/luci/import/src/Importer.test.cpp#L119

https://github.com/Samsung/ONE/blob/8180a7ce28f147ea43fa143840eb23c3d0f5708a/compiler/enco/core/src/Support/Debugging.h#L27